include/boost/multiprecision/detail/assert.hpp

```diff
@@ -0,0 +1,34 @@
+//  (C) Copyright Matt Borland 2021.
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// We deliberately use assert in here:
+//
+// boost-no-inspect
+
+#ifndef BOOST_MP_TOOLS_ASSERT_HPP
+#define BOOST_MP_TOOLS_ASSERT_HPP
+
+#include <boost/multiprecision/detail/standalone_config.hpp>
+
+#ifndef BOOST_MP_STANDALONE
+
+#include <boost/assert.hpp>
+#include <boost/static_assert.hpp>
+#define BOOST_MP_ASSERT(expr) BOOST_ASSERT(expr)
+#define BOOST_MP_ASSERT_MSG(expr, msg) BOOST_ASSERT_MSG(expr, msg)
+#define BOOST_MP_STATIC_ASSERT(expr) BOOST_STATIC_ASSERT(expr)
```

Comment:
I don't think these are in use any more - can we check and maybe remove all references to BOOST_STATIC_ASSERT?